spec/render_jsonapi_errors_spec.rb

```diff
@@ -17,7 +17,7 @@
     {
       'errors' => [
         {
-          'detail' => 'Name can\'t be blank',
+          'detail' => "Name #{I18n.t('errors.messages.blank')}",
```

Comment:
A couple of questions here.

- Was the lack of localization causing failures?
- I don't see a locale file that defines `errors.messages.blank`. Is this a bult-in from `I18n`? Should we start including a locale file to use in these tests?